GameServer/Logic/GameObjects/Monster.cs

```diff
@@ -16,6 +16,8 @@ public class Monster : Unit
         public byte CampUnk { get; private set; }
         public float SpawnAnimationTime { get; private set; }
 
+        private RAFManager _rafManager = Program.ResolveDependency<RAFManager>();
```

Comment:
Instead of adding this to every subclass, just add this to the `Unit` base class